tests/interface/TWPrivateKeyTests.cpp

```diff
@@ -73,11 +73,13 @@ TEST(PrivateKeyTests, ClearMemory) {
 
     TWPrivateKeyDelete(privateKey);
 
-    // Memory cleaned (filled with 0s).  They may be overwritten by something else; we check that it is not equal to original)
+    // Memory cleaned (filled with 0s).  They may be overwritten by something else; we check that it is not equal to original, most of it has changed.
     ASSERT_NE(privKey, TW::hex(TW::data(ptr, 32)));
+    int countDifferent = 0;
     for (auto i = 0; i < privKeyData.size(); ++i) {
-        ASSERT_NE(ptr[i], privKeyData[i]);
```

Comment:
is this case duplicated with `tests/PrivateKeyTests.cpp ` ?

Reply:
Yes, it's duplicated now, but in fact was duplicated from TWPrivateKeyTests.cpp, C++ tests were missing altogether.